src/javascripts/components/options-table.js

```diff
@@ -28,14 +28,12 @@ var OptionsTable = {
           var detailsText = optionsDetailsElement.querySelector('.govuk-details__text')
 
           if (detailsSummary && detailsText) {
-            tabLink.setAttribute('aria-expanded', true)
+            tabLink.setAttribute('aria-expanded', 'true')
             tabHeading.className += ' app-tabs__item--current'
-            tabsElement.classList.remove('app-tabs__container--hidden')
-            tabsElement.setAttribute('aria-hidden', false)
+            tabsElement.removeAttribute('hidden')
```

Comment:
This appears to be an issue with the existing JavaScript as well, but if you follow the link to the Nunjucks options from another component (e.g. https://deploy-preview-2242--govuk-design-system-preview.netlify.app/components/error-message/#options-error-message-example) on mobile you get a weird open state with a double border between the tab and the panel:

![deploy-preview-2242--govuk-design-system-preview netlify app_components_error-message_(iPhone 12 Pro)](https://user-images.githubusercontent.com/121939/179985704-54e10b21-0391-44da-a08b-503ba8be372d.png)

I think this is because the `app-tabs__heading--current` isn't being added to the tab?

Given this is an existing issue, if this isn't trivial to fix then let's leave it for now.

Reply:
Wasn't aware of that one specifically, but I did notice that the options-table.js file seems to only account for the desktop tabs. It'd be a nice candidate to for refactoring itself, especially if it becomes possible to toggle the details/tabs state programatically rather than setting classes/attributes manually.